metadata-ingestion/src/datahub/cli/docker_cli.py

```diff
@@ -901,28 +905,17 @@ def valid_restore_options(
     type=click.Path(exists=True, dir_okay=False),
     help=f"The MCE json file to ingest. Defaults to downloading {BOOTSTRAP_MCES_FILE} from GitHub",
 )
-@click.option(
-    "--token",
-    type=str,
-    is_flag=False,
-    default=None,
-    help="The token to be used when ingesting, used when datahub is deployed with METADATA_SERVICE_AUTH_ENABLED=true",
-)
 @telemetry.with_telemetry()
-def ingest_sample_data(path: Optional[str], token: Optional[str]) -> None:
+def ingest_sample_data(path: Optional[str]) -> None:
     """Ingest sample data into a running DataHub instance."""
 
+    test_connectivity_complain_exit("ingest-sample-data")
+
     if path is None:
         click.echo("Downloading sample data...")
         path = str(download_sample_data())
 
-    # Verify that docker is up.
-    status = check_docker_quickstart()
-    if not status.is_ok():
-        raise status.to_exception(
-            header="Docker is not ready:",
-            footer="Try running `datahub docker quickstart` first.",
```

Comment:
why are we removing this?

Reply:
Because that is code only for checking against localhost Quickstart deployments. The idea is to make ingest-sample-data work against remote datahub instances for which this logic makes little sense.

Reply:
ingest sample data is part of quickstart and we should keep it's scope limited

I have this PR which is related https://github.com/datahub-project/datahub/pull/7871 - that might be a better place for this stuff

Reply:
I was not aware of that, this PR is a follow up to https://github.com/datahub-project/datahub/pull/7896
I would personally like to have ingest-sample-data available to non-quick start deployments so that I can quickly provision data in a given deployment for testing.